packages/base/catalog-entry.gts

```diff
@@ -30,46 +33,12 @@ export class CatalogEntry extends CardDef {
       return new URL(this.ref.module, this[relativeTo]).href;
     },
   });
-  @field demo = contains(FieldDef);
```

Comment:
I removed this for now and will treat adding examples of different boxel spec in separate PR

Reply:
this actually needed a lot more discussion in terms of how we render this as it should be a card and not a field. but yeah, good to remove for now...